src/pages/Beta/Swap/WatchList/index.tsx

```diff
@@ -79,8 +79,9 @@ const WatchList: React.FC<Props> = ({ isLimitOrders }) => {
         </Box>
       </Box>
       <GridContainer isLimitOrders={isLimitOrders}>
-        {!isLimitOrders && (
+        {isLimitOrders && (
```

Comment:
any reason to revert this condition ?

Reply:
This is a mistake on our part, because we wanted to fix the display problem on the WAGMI testnet which does not have a dollar value, but it impacts the display of the mainnets. I will use the attribute is_mainnet instead in the new PR.